<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #ffffff;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>551</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Contact</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_website">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_website {
	background-color: #81d4f4;
	color: #ffffff;
	transition: 2s;
	border-radius: 6px;
}

#btn_website:hover {
	background-color: salmon;
}</string>
    </property>
    <property name="text">
     <string>Website</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>551</width>
      <height>192</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#textBrowser {
	border: 0;
}</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Jika ada yang salah atau ada yang kurang dari program ini silahkan kalian kirim ke kontak gw bro, dibawah gw dah sediain. Ada link website, yt, wa, dan juga blogger gw juga bro. Jangan lupa download juga program yang lainnya dari D'Startup dengan cara cek melalui website D'Startup atau cek di chanel yt D'Startup.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Jika ada yang kurang paham dengan cara penggunaan dari program ini silahkan kalian bisa baca melalui menu help atau japri gw bro.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;Terima kasih buat yang udah download program sederhana gw ini bro.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_yt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_yt {
	background-color: #81d4f4;
	color: #ffffff;
	transition: 2s;
	border-radius: 6px;
}

#btn_yt:hover {
	background-color: salmon;
}</string>
    </property>
    <property name="text">
     <string>YouTube</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_blogger">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_blogger {
	background-color: #81d4f4;
	color: #ffffff;
	transition: 2s;
	border-radius: 6px;
}

#btn_blogger:hover {
	background-color: salmon;
}</string>
    </property>
    <property name="text">
     <string>Blogger</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_wa">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>300</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_wa {
	background-color: #81d4f4;
	color: #ffffff;
	transition: 2s;
	border-radius: 6px;
}

#btn_wa:hover {
	background-color: salmon;
}</string>
    </property>
    <property name="text">
     <string>WhatsApp</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_email1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>260</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_email1 {
	background-color: #81d4f4;
	color: #ffffff;
	transition: 2s;
	border-radius: 6px;
}

#btn_email1:hover {
	background-color: salmon;
}</string>
    </property>
    <property name="text">
     <string>Email 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_email2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#btn_email2 {
	background-color: #81d4f4;
	color: #ffffff;
	transition: 2s;
	border-radius: 6px;
}

#btn_email2:hover {
	background-color: salmon;
}</string>
    </property>
    <property name="text">
     <string>Email 2</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
